### PR TITLE
Release 3.4.0

### PR DESCRIPTION
**Pull request by ndellingwood**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
 # Change Log
 
+## [3.4.00](https://github.com/kokkos/kokkos-kernels/tree/3.4.00) (2021-04-25)
+[Full Changelog](https://github.com/kokkos/kokkos-kernels/compare/3.3.01...3.4.00)
+
+**Features:**
+- SYCL: adding ETI and CMake logic for SYCL backend [\#924](https://github.com/kokkos/kokkos/pull/924)
```
Comment (lucbv) [inline CHANGELOG.md:7]: The link should really be: `https://github.com/kokkos/kokkos-kernels/pull/924`
The same is true for the other links below.

**Review by lucbv — COMMENTED** [review]
@ndellingwood there is some small issue with the change log.
It would be nice to fix it.

File: CHANGELOG.md (hunk 1 above)
Comment (lucbv) [reply CHANGELOG.md]: Actually I just did that in the commit below.

Comment (ndellingwood) [reply CHANGELOG.md]: Ack, good catch, I reused the same scripting from kokkos but forgot to update the link for kokkos-kernels

Comment (ndellingwood) [reply CHANGELOG.md]: @lucbv looks like you beat me to the changelog repair, thanks for the fix!

**Review by crtrott — APPROVED** [review]

**Pull request merged**